hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/ddl/HMSDDLExecutor.java

```diff
@@ -219,6 +219,7 @@ public void updatePartitionsToTable(String tableName, List<String> changedPartit
         sd.setLocation(fullPartitionPath);
         return new Partition(partitionValues, syncConfig.databaseName, tableName, 0, 0, sd, null);
       }).collect(Collectors.toList());
+      System.out.println("total partitions " +  partitionList.size());
```

Comment:
remove?

Reply:
Will move this to draft PR , it was created for initial collaboration, need more changes to fix some failing tests